Exception on index when running JEST

# Describe the bug

`index.ts` used `process.emit('SIGINT');` to gracefully exit on Windows, but this functionality was broken once Jest was added into the project. This resulted in the error: 

```
index.ts:123:32 - error TS2769: No overload matches this call.
      The last overload gave the following error.
        Argument of type '0' is not assignable to parameter of type 'Signals'.
```

However, the remaining files seemed to work correctly and the tests completed successfully once the line was commented out. 


## To Reproduce
Steps to reproduce the behavior:

1) Go to `index.ts` and uncomment the line from: 

```typescript
// Gracefully disconnect Postgres-Client on exit
if (process.platform === 'win32') {
    let rl = require('readline').createInterface({
        input: process.stdin,
        output: process.stdout
    });
    rl.on("SIGINT", () => {
        // process.emit('SIGINT', 0);
        process.exit();
    });
}
```

2) Run the tests in the terminal with `npm test`


## Expected behavior

This code worked correctly before Jest and dependencies were added, so the tests should run successfully. The pokebot itself should also function with this line and exit gracefully in windows. 

## Possible solutions
The function seems to expect a `Signals` type parameter so more research needs to be done into this function in typescript to see if there was an error in the code.

--- a/.github/workflows/jest.yml
+++ b/.github/workflows/jest.yml
@@ -1,0 +1,18 @@
+name: Jest Test
+
+on:
+  push:
+    branches: [ "master", "develop" ]
+  pull_request:
+    branches: [ "master", "develop" ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v3
+    - name: Install dependencies
+      run: npm i
+    - name: Test using jest
+      run: npm test

--- a/index.ts
+++ b/index.ts
@@ -120,7 +120,6 @@ if (process.platform === 'win32') {
         output: process.stdout
     });
     rl.on("SIGINT", () => {
-        // process.emit('SIGINT', 0);
         process.exit();
     });
 }

--- a/package.json
+++ b/package.json
@@ -21,5 +21,17 @@
     "@jest/globals": "^29.4.3",
     "jest": "^29.4.3",
     "ts-jest": "^29.0.5"
+  },
+  "jest": {
+    "verbose": true,
+    "moduleFileExtensions": ["js", "jsx", "json", "ts", "tsx"],
+    "roots": ["<rootDir>", "<rootDir>/test"],
+    "transform": {
+      "^.+\\.m?[tj]sx?$": [
+        "ts-jest",
+        {
+        }
+      ]
+    }
   }
 }

--- a/test/index.test.ts
+++ b/test/index.test.ts
@@ -1,1 +1,9 @@
 import { describe, expect, test } from '@jest/globals';
+
+function sum(a: number, b: number) {
+  return a + b;
+}
+
+test('adds 1 + 2 to equal 3', () => {
+  expect(sum(1, 2)).toBe(3);
+});

